### DISCUSSION TITLE
Adding a daily limit for season tickets missions

### DISCUSSION THREAD

**Discussion by UcanMessiJesus** [sunflower-land/sunflower-land#2470] [Ideas]
Hello new season is approaching

But I think there is a problem

People who constantly follow the game and work at the computer can follow the seasonal tasks very easily, but for people who work hard or do not work at the computer, it becomes difficult to do seasonal tasks.

As a working person, I look at the game during my work breaks and occasionally by sneaking around. This problem causes seasonal tasks to progress less daily, while someone else can do 4-5 tasks a day, I can hardly complete 2 tasks a day.

Do you have a solution for this problem?

**Comment by offerT** [comment]
This is unfair to players who have been playing for a long time. Don't be so selfish
